databricks_cli/sdk/service.py

```diff
@@ -522,3 +541,83 @@ def get_status(self, path):
         return self.client.perform_query('GET', '/workspace/get-status', data=_data)
      
 
+class SecretService(object):
+    def __init__(self, client):
+        self.client = client
+
+    def create_scope(self, scope, initial_manage_acl=None):
+        _data = {}
+        if scope is not None:
+            _data['scope'] = scope
+        if initial_manage_acl is not None:
+            _data['initial_manage_acl'] = initial_manage_acl
+        return self.client.perform_query('POST', '/secret/scopes/create', data=_data)
+    
+    def delete_scope(self, scope):
+        _data = {}
+        if scope is not None:
+            _data['scope'] = scope
+        return self.client.perform_query('POST', '/secret/scopes/delete', data=_data)
+    
+    def list_scopes(self):
+        _data = {}
+    
```

Comment:
nit: extra newline

Reply:
Maybe we should keep it in order to be consistent with the other parts, as it's autogenerated